### ISSUE TITLE
Changing the project state on the fly

### ISSUE THREAD

**Issue by vimlesh1975** [theatre-js/theatre#389]
I am making a project which animates fabricjs objects with theatrejs studio, and exports pure html with fabirc.min.js and core-only.min.js having animation state..
While working on a project, I would like to import already exported file to modify. I can remove existing fabric objects and sheet objects, add imported fabric objects and create sheet objects. Up to this is fine.
But sequence state is not being applied.
basically I want to call again project = getProject('HTML Animation Tutorial', {state}), but this would be a new state. But this is disallowed.
I want to ask what is workaround for this use case?
https://vimlesh1975.github.io/ReactCasparClient/WebAnimator

**Comment by vimlesh1975** [comment]
Now as a workaround I am creating a new project while importing a file. Now I want to delete earlier project and its sheet.

**Comment by GhostCatcg** [comment]
Hello, have you found any way to solve the problem of not being able to import again?
